src/components/Login/LoginForm.tsx

```diff
@@ -44,10 +47,14 @@ const LoginForm = ({ addNewWallet }: LoginFormProps) => {
   const [feedback, setFeedback] = useState('');
   const [showSpinner, setShowSpinner] = useState(false);
 
-  const handleKeyDown = useCallback(
-    e => e.key === 'Enter' && dispatch.account.login({ key: loginValue, setError, setFeedback }),
-    [loginValue, setError]
-  );
+  const performLogin = useCallback(() => {
+    if (!loginValue || loginValue === '') {
```

Comment:
Empty strings are falsy in JS so the OR part of this statement is probably unnecessary.